app/preview/TemplateParams.scala

```diff
@@ -2118,10 +2026,6 @@ object TemplateParams2 {
                   "message": "Value must be a key-value data structure",
                   "percentage": 10,
                   "count": 1075
-                }, {
```

Comment:
Had to remove a validation message as adding the two new params meant that the URL for previewing the template became too long